members/admin/__init__.py

```diff
@@ -258,9 +263,9 @@ def queryset(self, request, queryset):
             )
 
 
-class ActivityParticipantListFilter(admin.SimpleListFilter):
+class ActivityParticipantListOldYearsFilter(admin.SimpleListFilter):
     # Title shown in filter view
-    title = "Efter aktivitet (før år " + str(timezone.now().year - 1) + ")"
+    title = "Efter aktivitet (\u2264 år " + str(timezone.now().year - 2) + ")"
```

Comment:
En kommentar ovenover, der siger hvad der står her ville være lækkert, da jeg i hvert fald ikke i hovedet kan huske hvad u2264 er :slightly_smiling_face:

Reply:
ok, er rettet
    # Title shown in filter view. \u2264 : ≤ (mindre end eller lig med)